source/index.txt

```diff
@@ -20,7 +20,7 @@ What are the Atlas Application Services?
    Introduction </introduction>
    Tutorials </tutorials>
    Device Sync </sync>
-   Edge Server </edge-server>
+   Edge Server Preview </edge-server>
```

Comment:
I don't know how I feel about this, but should Preview here be in parens? i.e.:

```suggestion
   Edge Server (Preview) </edge-server>
```

I think the fact that it's in Preview is secondary, but having it _not_ in parens makes me think this is just some temp thing and the "real" thing will be different somehow. (Which is maybe what we want to communicate? Or maybe I'm just overthinking it.)